code/modules/games/tarot.dm

```diff
@@ -168,3 +169,220 @@
+				mystery, ancestral voices."
+			if("Windless Day")
+				P.desc = "A completely still scene, a frozen lake, unmoving trees, a bird suspended in midair. A sense of being trapped in time. A small crack in the lake surface forms the glyph for 'waiting'. It is associated with: \
+				stagnation, hidden tension."
+			if("Black Gale")
+				P.desc = "A storm-black wind crashes into tents, tearing them apart as lightning strikes nearby. The wind has claws. Each lightning bolt forms a different rune for destruction, punishment. It is associated with: \
+				upheaval, divine punishment."
+			if("Whisper Breeze")
+				P.desc = "A soft breeze curls around a sleeping child’s ear, one candle stands nearby. It is associated with: secrets, subtle shifts."
+			if("Stormcry")
+				P.desc = "Rain and wind mix with wailing spirit-faces. A lone figure kneels on a clifftop, arms raised in anguish. Tears fall upward, the wind feels heavy. It is associated with: rage, grief, release."
+		P.card_icon = "winds"
+		P.back_icon = "card_back_lyodii"
+		cards += P
+
+/obj/item/deck/tarot/lyodii/bones
+	name = "lyodii fatesayer bones deck"
+	desc = "A traditionally made deck of fatesayer cards, used by the people of the Lyod. This stack contains the Bones cards. "
+	icon_state = "lyodii_deck"
+	item_state = "lyodii_deck"
+
+/obj/item/deck/tarot/lyodii/bones/generate_deck()
+	var/datum/playingcard/P
+	for(var/name in list("Bone of Birth","Bone of Death","Bone of Choice","Goddess"))
+		P = new()
+		P.name = "[name]"
+		P.card_icon = "bones"
+		P.back_icon = "card_back_lyodii"
+		switch(name)
+			if("Bone of Birth")
+				P.desc = "A curved rib bone floats in a pool of water, illuminated from below. Inside it, a tiny sprout grows. In the water’s reflection, the bone looks like a cradle. It is associated with: legacy, fate beginning to move."
+			if("Bone of Death")
+				P.desc = "A skull, half-buried in snow, with a single red flower growing from one eye socket. A spirit form rises behind the skull, birds fly overhead. It is associated with: inevitable ends, sacred cycles."
+			if("Bone of Choice")
+				P.desc = "A forked femur lies between two paths—one lined with fire, the other shadowy. A hand hesitates above it. The bone is marked with tally marks and blood-stained glyphs. It is associated with: the moment of truth, \
+				taking action."
+			if("Goddess")
+				P.desc = "A glowing, diffuse spectral form floats above a ritual circle of bones. Light spills from the heavens onto it. Snowflakes falling nearby take on different shapes of animals, eyes and hands. It is associated with: \
+				divine will, judgement, revelation."
+				P.card_icon = "goddess"
+		cards += P
+
+/obj/item/storage/box/lyodii
+	name = "fatesayer box"
+	desc = "A small leather case to to hold all 36 cards of a Fatesayer deck."
+	icon_state = "card_holder_empty"
+	icon = 'icons/obj/storage/misc.dmi'
+	can_hold = list(/obj/item/deck/tarot/lyodii, /obj/item/hand, /obj/item/card)
```

Comment:
Does this need to hold hands or cards? This would allow standard cards to be held in this case, but not standard card decks. May be a slight issue there.

Reply:
Right. Good spot. At first I intended it to only hold the new tarot cards but thinking about it, it didnt make much sense to me. This should hold *decks*, not hands. In the end it is just a vessel to spawn all five decks in, nothing more.